sdk/go/auto/stack_test.go

```diff
@@ -0,0 +1,35 @@
+package auto
```

Comment:
This file will need the [copyright preamble](https://github.com/pulumi/pulumi/blob/master/sdk/go/auto/local_workspace_test.go#L1-L13), or you can put this test in `local_workspace_test.go` - up to you :)

Reply:
@komalali I added a modified version using the current year and fixed the conflicts, PTAL

Reply:
Perfect, thank you!